Don't set this for gcc, it keeps snappy from building

Index: snappy-stubs-internal.h
--- snappy-stubs-internal.h.orig
+++ snappy-stubs-internal.h
@@ -99,7 +99,7 @@
 #endif
 
 // Inlining hints.
-#ifdef HAVE_ATTRIBUTE_ALWAYS_INLINE
+#if defined(HAVE_ATTRIBUTE_ALWAYS_INLINE) && defined(__clang__)
 #define SNAPPY_ATTRIBUTE_ALWAYS_INLINE __attribute__((always_inline))
 #else
 #define SNAPPY_ATTRIBUTE_ALWAYS_INLINE
